policy/common/result.go

```diff
@@ -59,12 +59,20 @@ type ReviewRequestRule struct {
 	Mode RequestMode
 }
 
+type Requires struct {
+	Count int `yaml:"count"`
+
+	Actors `yaml:",inline"`
+}
+
 type Result struct {
 	Name              string
 	Description       string
 	StatusDescription string
```

Comment:
It is hard to combine to existing structs because `PredicatesInfo` is `[]*PredicateInfo` and `Requires` is `Actors`. On the other hand, I think creating another struct to combine all three is kind of unnecessary. So I hold my opinion on this one and thinks it is fine the way it is now.